clients/cli/src/main/java/org/apache/gravitino/cli/TagCommandHandler.java

```diff
@@ -0,0 +1,207 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *  http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+package org.apache.gravitino.cli;
+
+import java.util.Arrays;
+import org.apache.commons.cli.CommandLine;
+import org.apache.gravitino.cli.commands.Command;
+
+public class TagCommandHandler extends CommandHandler {
+  private final GravitinoCommandLine gravitinoCommandLine;
+  private final CommandLine line;
+  private final String command;
+  private final boolean ignore;
+  private final String url;
+  private final String[] tags;
+  private String metalake;
+
+  public TagCommandHandler(
+      GravitinoCommandLine gravitinoCommandLine, CommandLine line, String command, boolean ignore) {
+    this.gravitinoCommandLine = gravitinoCommandLine;
+    this.line = line;
+    this.command = command;
+    this.ignore = ignore;
+    this.url = getUrl(line);
+    String[] tagFromCommandLine = line.getOptionValues(GravitinoOptions.TAG);
+    this.tags =
+        tagFromCommandLine == null
+            ? null
+            : Arrays.stream(tagFromCommandLine).distinct().toArray(String[]::new);
+  }
```

Comment:
This is more complex and harder to understand than the original code - why change it?

Reply:
I want to keep the `final` keyword and provide some protection from accident changing reference.
It just my personal taste. 
WDYT ?

Reply:
I think calling "tagFromCommandLine" is a little misleading, overly verbose and could confuse the reader of the code. It can include more than one tag so the word tag is incorrect. It also doesn't fit the style of the rest of the code.

Reply:
I see.
I will update it soon 😺

Reply:
The code is also more complex, and the intent is less clear, it's better to keep it easily understandable. I appreciate you trying to clean up some things while refactoring but it's best to keep it as it was.